scalafix/rules/src/main/scala/fix/v0_14_0/FixAvroCoder.scala

```diff
@@ -162,9 +165,23 @@ object FixAvroCoder {
           coderT
       }
       .flatMap(_.info.map(_.signature).toList)
-      .exists { case TypeSignature(_, _, TypeRef(_, maybeAvroType, _)) =>
-        AvroMatcher.matches(maybeAvroType)
+      .exists {
+        case TypeSignature(_, _, TypeRef(_, maybeAvroType, _)) =>
+          AvroMatcher.matches(maybeAvroType)
+        case _ => false
```

Comment:
I was getting a compiler warning about this line, so added the exhaustive matcher